src/Directives.jl

```diff
@@ -95,11 +99,20 @@ function parsebracket end
 parsebracket(text :: AbstractString) = parsebracket(getdirective(text)...)
 parsebracket{D}(:: Directive{D}, text) = error("Unknown directive: '$D'")
 
+function parsebracket{D}(TD :: Directive{D}, arg, text)
+    if strip(arg) == ""
+        parsebracket(TD, text)
+    else
+        error("parsebracket not defined for directive '$D' with positional argument.")
+    end
+end
+
 function parsebracket(:: directive"docs", text)
+    eval(Expr(:toplevel, Expr(:using, :Lexicon)))
```

Comment:
Building particular files outside of ``build`` will need some kind of syntax or option passed to ``makedocs``. Perhaps:

```jl
makedocs(
    external = ("README.md" => "../README.md",)
)
```

where each pair represents the file to treat differently and it's destination (relative to the ``doc`` directory). Not sure how I feel about that syntax and naming though.